packages/blip/app/components/chart/trendssubnav.js

```diff
@@ -202,8 +202,10 @@ class TrendsSubNav extends React.Component {
       'inactive': !this.props.activeDays[day]
     }) + ' ' + day
 
+    const dayFilterId = `day-filter-${day}`
+
     return (
-      <a className={dayLinkClass} key={day} onClick={this.props.onClickDay(day)}>{this.renderDayAbbrev(day)}</a>
+      <a data-testid={dayFilterId} className={dayLinkClass} key={day} onClick={this.props.onClickDay(day)}>{this.renderDayAbbrev(day)}</a>
```

Comment:
I understand why sonar isn't happy and it is symptomatic of another issue: I don't think that `<a>` elements should be used like this (see [here](https://developer.mozilla.org/fr/docs/Web/HTML/Element/a)).
Here, what we should use is [this](https://mui.com/material-ui/react-chip/) (basically, some kind of toggle)